jobs/config.json

```diff
@@ -12038,7 +12038,7 @@
       "--gke-environment=prod",
       "--provider=gke",
       "--test=false",
-      "--test-cmd=demos/integration_tests.sh",
+      "--test-cmd=./demos/integration_tests.sh",
```

Comment:
I'm pretty sure this is indeed the necessary fix. 🤞

Reply:
gah, it's `../demos/integration_test.sh` because kubetest also is upping the cluster for a `kubernetes` directory and staying there once it starts the tests ... >< @krzyzacy this definitely makes kubetest more annoying to use :(

> I0517 04:35:08.339] Extracting kubernetes-test.tar.gz into /go/src/github.com/kubernetes-sigs/kustomize/kubernetes


I kicked off a run with the full path (added `--test-cmd=/go/src/github.com/kubernetes-sigs/kustomize/demos/integration_tests.sh` to the end of the args), and there's another issue -- the kustomize script will fail if it's not run from the root of the repo.

Reply:
(https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kustomize-periodic-default-gke/7/)